packages/system/kcrypt/build.yaml

```diff
@@ -1,7 +1,15 @@
+{{if eq .Values.category "fips" }}
+requires:
+  - name: "toolchain-go-ubuntu"
+    category: "fips"
```

Comment:
is there no way to inherit the category?

Reply:
oh I see, good point I can just use `{{.Values.category}}` here! :facepalm: